inspirehep/base/templates/format/record/Inspire_Default_HTML_brief_macros.tpl

```diff
@@ -18,8 +18,7 @@
 #}
 
 {% macro render_doi() %}
-  {% if record.get('dois')| is_list() %}
-    {% set filtered_doi = record.get('dois.value')| remove_duplicates() %}
+    {% set filtered_doi = record.get('dois.value')|tranform_nested_list()|remove_duplicates() %}
```

Comment:
so what you are trying to do here is to flatten a nested list (http://stackoverflow.com/questions/952914/making-a-flat-list-out-of-list-of-lists-in-python) - you could rename the filter.

But the question is, why do you get a list of lists? I don't see in the `dojson` definition when that would be the case - https://github.com/inspirehep/inspire-next/blob/master/inspirehep/dojson/hep/fields/bd01x09x.py#L53

It might be better to solve the source of the problem. You can run the problematic record though `dojson` and see what is going on.